### PR TITLE
Add PMGSY GeoSadak 2022 (India)

### PR DESCRIPTION
**Pull request by Vonter**
Layer information from https://github.com/datameet/pmgsy-geosadak

### CODE REVIEW

**Comment by planemad** [comment]
The also includes OSM data and Mapbox Satellite imagery, which overall should be fine for tracing on OSM, but not for other uses. We should correct the license to say ODbL to make this clear.

If the tiles have only the Geosadak data, that would make it a truly CC-by / https://data.gov.in/Godl level tile layer, but we could do that in the future.

Let me also switch the token with one that is specific for OSM India Community use.

File: sources/asia/in/India-PMGSY.geojson [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+    "properties": {
+        "id": "India-PMGSY",
+        "attribution": {
+            "url": "https://geosadak-pmgsy.nic.in/OpenData",
+            "text": "PMGSY National GIS - Open Data",
+            "required": true
+        },
+        "name": "PMGSY GeoSadak",
+        "type": "tms",
+        "url": "https://api.mapbox.com/styles/v1/planemad/cknj1leps0ywv17lrj8d16vnj/tiles/256/{zoom}/{x}/{y}?access_token=pk.eyJ1IjoicGxhbmVtYWQiLCJhIjoiY2t6enBzbzdsMGNrZzNjbXhyeWdkeW55MCJ9.tAP01enwlIZkOuQth0mJLA",
```
Comment (planemad) [inline sources/asia/in/India-PMGSY.geojson]:
```suggestion
        "url": "https://api.mapbox.com/styles/v1/planemad/cknj1leps0ywv17lrj8d16vnj/tiles/256/{zoom}/{x}/{y}?access_token=pk.eyJ1Ijoib3NtaW5kaWEiLCJhIjoiY2x2M2h0dmtnMDB5MzJqbzBsNmhwczJubSJ9.XE-a2USKZUYH4t8rTFEnmg",
```

Comment (planemad) [reply sources/asia/in/India-PMGSY.geojson]: Token from the Mapbox `osmindia` account for OSM community use

File: sources/asia/in/India-PMGSY.geojson [hunk 2]
```diff
@@ -0,0 +1,432 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "India-PMGSY",
+        "attribution": {
+            "url": "https://geosadak-pmgsy.nic.in/OpenData",
+            "text": "PMGSY National GIS - Open Data",
+            "required": true
+        },
+        "name": "PMGSY GeoSadak",
```
Comment (planemad) [inline sources/asia/in/India-PMGSY.geojson]:
```suggestion
        "name": "PMGSY GeoSadak - India Rural Roads Open Data",
```

Comment (planemad) [reply sources/asia/in/India-PMGSY.geojson]: More informative name for someone unaware of PMGSY

File: sources/asia/in/India-PMGSY.geojson [hunk 3]
```diff
@@ -0,0 +1,432 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "India-PMGSY",
+        "attribution": {
+            "url": "https://geosadak-pmgsy.nic.in/OpenData",
+            "text": "PMGSY National GIS - Open Data",
```
Comment (planemad) [inline sources/asia/in/India-PMGSY.geojson:7]:
```suggestion
            "text": "PMGSY National GIS - Open Data",
            "html": "<a href='https://geosadak-pmgsy.nic.in/opendata/' target='_blank'>PMGSY Rural Connectivity Dataset, Government Open Data License</a> | <a href='https://www.mapbox.com/about/maps/' target='_blank' title='Mapbox' aria-label='Mapbox'>© Mapbox</a> | <a href='https://www.openstreetmap.org/about/' target='_blank' title='OpenStreetMap' aria-label='OpenStreetMap'>© OpenStreetMap</a> | <a href='https://www.maxar.com/' target='_blank' title='Maxar' aria-label='Maxar'>© Maxar</a>",
```

Comment (planemad) [reply sources/asia/in/India-PMGSY.geojson:7]: Making sure all sources are attributed

**Review by planemad — CHANGES_REQUESTED** [review]

**Comment by Vonter** [comment]
Thanks a lot for the suggestions @planemad, I have included them in the JSON.

**Comment by Vonter** [comment]
@planemad it looks like the OSM India Community token fails to load the tiles due to an access forbidden 403 error. I have reverted back to the original token for now, but if there's a way to fix the access with the new token, I could try it out.

File: sources/asia/in/India-PMGSY.geojson [hunk 4]
```diff
@@ -0,0 +1,435 @@
...
+            "html": "<a href='https://geosadak-pmgsy.nic.in/opendata/' target='_blank'>PMGSY Rural Connectivity Dataset, Government Open Data License</a> | <a href='https://www.mapbox.com/about/maps/' target='_blank' title='Mapbox' aria-label='Mapbox'>© Mapbox</a> | <a href='https://www.openstreetmap.org/about/' target='_blank' title='OpenStreetMap' aria-label='OpenStreetMap'>© OpenStreetMap</a> | <a href='https://www.maxar.com/' target='_blank' title='Maxar' aria-label='Maxar'>© Maxar</a>",
+            "required": true
+        },
+        "name": "PMGSY GeoSadak - India Rural Roads Open Data",
+        "type": "wmts",
+        "url": "https://api.mapbox.com/styles/v1/planemad/cknj1leps0ywv17lrj8d16vnj/wmts?access_token=pk.eyJ1IjoicGxhbmVtYWQiLCJhIjoiY2l2eDhnNzNpMDAwNzJ5cGowcnpiMXJkdyJ9.NljuPglsRA3mTGf-4CLIEg",
```
Comment (planemad) [inline sources/asia/in/India-PMGSY.geojson]:
```suggestion
        "url": "https://api.mapbox.com/styles/v1/planemad/cknj1leps0ywv17lrj8d16vnj/wmts?access_token=pk.eyJ1Ijoib3NtaW5kaWEiLCJhIjoiY2x4bm44YTVsMDR5bzJyczY5MjBoa2Q5aSJ9.r1MPugHw49L3kViEaWhBWA",
```

Comment (planemad) [reply sources/asia/in/India-PMGSY.geojson]: @Vonter fixed the token after testing

Comment (Vonter) [reply sources/asia/in/India-PMGSY.geojson]: Thanks! Works perfectly now :+1:

**Pull request merged**